ex1_bioblend_answers.ipynb

```diff
@@ -339,7 +347,522 @@
+      "                           'name': 'samtools_sort',\n",
+      "                           'owner': 'devteam',\n",
+      "                           'tool_shed': 'toolshed.g2.bx.psu.edu'},\n",
+      "  'version': '2.0.1',\n",
+      "  'xrefs': []},\n",
+      " {'description': 'data in ascending or descending order',\n",
+      "  'edam_operations': [],\n",
+      "  'edam_topics': [],\n",
+      "  'form_style': 'regular',\n",
+      "  'hidden': '',\n",
+      "  'id': 'toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sort_header_tool/1.1.0',\n",
+      "  'is_workflow_compatible': True,\n",
+      "  'labels': [],\n",
+      "  'link': '/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sort_header_tool%2F1.1.0',\n",
+      "  'min_width': -1,\n",
+      "  'model_class': 'Tool',\n",
+      "  'name': 'Sort',\n",
+      "  'panel_section_id': 'text_manipulation',\n",
+      "  'panel_section_name': 'Text Manipulation',\n",
+      "  'target': 'galaxy_main',\n",
+      "  'tool_shed_repository': {'changeset_revision': 'e39fceb6ab85',\n",
+      "                           'name': 'text_processing',\n",
+      "                           'owner': 'bgruening',\n",
+      "                           'tool_shed': 'toolshed.g2.bx.psu.edu'},\n",
+      "  'version': '1.1.0',\n",
+      "  'xrefs': []},\n",
+      " {'description': 'data in ascending or descending order',\n",
+      "  'edam_operations': [],\n",
+      "  'edam_topics': [],\n",
+      "  'form_style': 'regular',\n",
+      "  'hidden': '',\n",
+      "  'id': 'toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_sort_header_tool/1.1.1',\n",
+      "  'is_workflow_compatible': True,\n",
+      "  'labels': [],\n",
+      "  'link': '/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Ftext_processing%2Ftp_sort_header_tool%2F1.1.1',\n",
+      "  'min_width': -1,\n",
+      "  'model_class': 'Tool',\n",
+      "  'name': 'Sort',\n",
+      "  'panel_section_id': 'text_manipulation',\n",
+      "  'panel_section_name': 'Text Manipulation',\n",
+      "  'target': 'galaxy_main',\n",
+      "  'tool_shed_repository': {'changeset_revision': '74a8bef53a00',\n",
+      "                           'name': 'text_processing',\n",
+      "                           'owner': 'bgruening',\n",
+      "                           'tool_shed': 'toolshed.g2.bx.psu.edu'},\n",
+      "  'version': '1.1.1',\n",
+      "  'xrefs': []}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "pprint(tc.get_tools(name='Sort'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sort_tool = 'sort1'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "11) Check the tool inputs by calling `show_tool()` with `io_details=True`. It might be useful to open Galaxy and compare the inputs you see here to the tool inputs in the Galaxy UI."
```

Comment:
```suggestion
    "11) Inspect the tool inputs by calling `show_tool()` with `io_details=True`. It might be useful to open Galaxy and compare the inputs you see here to the tool inputs in the Galaxy UI."
```